test/e2e-go/upgrades/send_receive_upgrade_test.go

```diff
@@ -117,6 +117,10 @@ func generateFastUpgradeConsensus() (fastUpgradeProtocols config.ConsensusProtoc
 		}
 
 		fastUpgradeProtocols[consensusTestFastUpgrade(proto)] = fastParams
+
+		// support the ALGOSMALLLAMBDAMSEC = 500 env variable
+		fastParams.AgreementFilterTimeout = time.Second
```

Comment:
as before - if you've set this, you should be able to get rid of the `os.Setenv("ALGOSMALLLAMBDAMSEC", "500")`.
make sure that the various tests still takes have the same execution time.